src/app/shared-module/services/task.service.ts

```diff
@@ -0,0 +1,87 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Task } from '../types/task';
+
+@Injectable({ providedIn: 'root' })
+export class TaskService {
+  constructor(private http: HttpClient) {}
+
+  getTodayTasks(): Observable<Task[]> {
+    return new Observable((observer) => {
```

Comment:
you can use of(this.getTasks()) instead of creating. but it just mocks - doesn't really matter